.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,7 +8,7 @@ assignees: ''
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of what the problem is. Ex. When doing X, we are unable to Y, because [...]
```

Comment:
"unable to Y" may be too high of a bar.  Maybe the customer is looking for "it would be easier to do Y if the libraries did X".

Reply:
I like that, thanks @coryan